src/components/UpdatePaymentDetailsPopup/Steps/Success.js

```diff
@@ -23,20 +26,17 @@ const Success = ({ hideInnerPopup }) => {
         <TextStyled>{t('Your payment details have been updated')}</TextStyled>
       </ContentStyled>
       <ButtonWrapperStyled removeMargin>
-        <Button theme="simple" onClickFn={() => hideInnerPopup()}>
+        <Button
+          theme="simple"
+          onClickFn={() =>
+            dispatch(updatePaymentDetailsPopup({ isOpen: false }))
```

Comment:
just wondering if we should 'reset' more of the state here. What if someone immediately opens the popup once again?